cli/coverage_test.go

```diff
@@ -1,9 +1,31 @@
 package cli
 
 import (
+	"io/ioutil"
 	"testing"
 )
 
+func TestStripComments(t *testing.T) {
+
+	source, _ := ioutil.ReadFile("testdata/strip-comments/commented-source.txt")
+	target, _ := ioutil.ReadFile("testdata/strip-comments/expected-source.txt")
```

Comment:
You can just inline the text instead of storing them as separate files. Just use the Go string literal syntax:

```
str := `this
is a
multi line 
string literal`

```